pgxn/neon/walproposer_pg.c

```diff
@@ -220,6 +220,58 @@ nwp_register_gucs(void)
 							NULL, NULL, NULL);
 }
 
+
+static int
+split_safekeepers_list(char *safekeepers_list, char *safekeepers[])
+{
+	int n_safekeepers = -1;
+
+	for (char *s = strtok(safekeepers_list, ","); s != NULL; s = strtok(NULL, ","))
```

Comment:
`strchr` seems to be more appreciate here and it doesn't;t require copying of string

Reply:
Changed to `strchr` and added a test, which fails in main